Update ideas forum link (#1886)
OfficeDev/office-js: .github/ISSUE_TEMPLATE/feature-request.md

---
name: Feature request
about: To request a new feature in office.js or any other aspect of the Office developer

>   platform, post your request on the  Microsoft 365 Developer Platform Ideas Forum

Reviewer: ```suggestion
  platform, post your request on the Microsoft 365 Developer Platform Ideas Forum
```

title: ''
labels: ''
assignees: ''

---

> ## New Microsoft 365 Developer Platform Ideas forum

Reviewer: ```suggestion
## Microsoft 365 Developer Platform Ideas forum
```

Author: Fixed

> [!NOTE]

> > We will be moving away from UserVoice feedback sites on a product-by-product basis throughout the 2021 calendar year. We will leverage 1st party solutions for customer feedback. For more information, see [UserVoice pages](https://support.microsoft.com/topic/uservoice-pages-430e1a78-e016-472a-a10f-dc2a3df3450a).

Reviewer: ```suggestion
> We are moving away from UserVoice feedback sites on a product-by-product basis throughout the 2021 calendar year. We are leveraging Microsoft solutions for customer feedback. For more information, see [UserVoice pages](https://support.microsoft.com/topic/uservoice-pages-430e1a78-e016-472a-a10f-dc2a3df3450a).
```

Author: Fixed

> Post feature requests and suggestions at the [Microsoft 365 Developer Platform Ideas Forum](https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform)

Reviewer: ```suggestion
Post feature requests and suggestions at the [Microsoft 365 Developer Platform Ideas Forum](https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform).
```

Author: Reworded and fixed.
